module/proto/gravity/v1/msgs.proto

```diff
@@ -235,3 +239,13 @@ message MsgCancelSendToEth {
 }
 
 message MsgCancelSendToEthResponse {}
+
+// This call allows anyone to submit evidence that a
+// validator has signed a valset, batch, or logic call that never
+// existed. Subject contains the batch, valset, or logic call.
+message MsgSubmitBadSignatureEvidence {
+  google.protobuf.Any subject   = 1;
+  string              signature = 2;
+}
```

Comment:
rename to `MsgSubmitMisbehavior`, `subject` is also a bad term here and should be included in the misbehavior field. This msg should also contain the signer account address

Reply:
> rename to MsgSubmitMisbehavior

Disagree. "Misbehavior" is very ambiguous. I think there's definitely a better term than "BadSignatureEvidence", but that's not it.

Reply:
> This msg should also contain the signer account address

Ethereum account? Cosmos account?

Neither of these is required. The ethereum address is contained in the signature and is all that is needed to look up the validator. I don't think it's a good practice to bloat out the code with boilerplate that is not actually used by the code.

Reply:
Why not use MsgSubmitEvidence and use any to describe the type of evidence. This will allow for future iterations of the software to quickly add new types of evidence instead of doing a refactor. 

The design that's being taken could use a design doc, something similar to an ADR.